### DISCUSSION TITLE
[Cloud Engineer] Optimize impact on customer devices and equipment

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/patterns#37] [Ideas]
https://docs.aws.amazon.com/wellarchitected/latest/sustainability-pillar/optimize-impact-on-customer-devices-and-equipment.html

Intent: Understand the devices and equipment your customers use to consume your services, their expected lifecycle, and the financial and sustainability impact of replacing those components. Implement software patterns and architectures to minimize the need for customers to replace devices and upgrade equipment. For example, implement new features using code that is backwards compatible with older hardware and operating system versions, or manage the size of payloads so they don’t exceed the storage capacity of the target device.

**Comment by holanita** [comment]
[Issue created](https://github.com/Green-Software-Foundation/green-software-patterns/issues/56)
